bika/lims/browser/worksheet/views/analyses_transposed.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, bika_listing=None, table_only=True):
     def _transpose_data(self):
         cached = []
         index = 0
-        include = ['Attachments', 'DetectionLimit', 'DueDate', 'Pos']
+        include = ['Attachments', 'DetectionLimit', 'DueDate', 'Pos', 'Remarks']
```

Comment:
If you remove 'Remarks' from the "inline" view, remember you'll need to add the column in the transposed view.